be/src/common/status.h

```diff
@@ -327,7 +327,7 @@ class STATUS_ATTRIBUTE Status {
 
     /// @return A string representation of this status suitable for printing.
     ///   Returns the string "OK" for success.
-    std::string to_string() const;
+    std::string to_string(bool with_context_info = true) const;
```

Comment:
Please add UT with with_context_info enabled and disabled.

Reply:
OK